pwiz_tools/Skyline/TestPerf/PerfCommandlineCreateImsDbTest.cs

```diff
@@ -43,55 +50,78 @@ public void CommandlineCreateImsDbPerfTest()
+                    // Expect failure because of illegal characters in imsdb filename
+                    AssertEx.Contains(output, 
+                        string.Format(Resources.ValueInvalidPathException_ValueInvalidPathException_The_value___0___is_not_valid_for_the_argument__1__failed_attempting_to_convert_it_to_a_full_file_path_,
+                            imsdb, "--ionmobility-library-create"));
+                    continue;
+                }
+                else if (Equals(libName, badPath))
+                {
+                    // Expect failure because of nonexistent subdirectories in path
+                    AssertEx.ContainsSimilar(output, Resources.CommandLine_SaveFile_Error__The_file_could_not_be_saved_to__0____Check_that_the_directory_exists_and_is_not_read_only_);
+                    continue;
+                }
+                else if (Equals(libName, badArgs))
+                {
+                    // Expect failure because of missing "--ionmobility-library-create"
+                    AssertEx.Contains(output, 
+                        string.Format(Resources.CommandArgs_WarnArgRequirment_Warning__Use_of_the_argument__0__requires_the_argument__1_,
+                            "--ionmobility-library-name", "--ionmobility-library-create"));
                     continue;
                 }
                 AssertEx.FileExists(imsdb);
-                AssertEx.Contains(output, $"Ion mobility library changed from \"None\" to \"{(string.IsNullOrEmpty(libName) ? root : libName)}\"");
 
-                // Compare to expected result
+                // Compare to expected result - may need to localize the expected copy to match the actual copy
                 var expected = File.ReadAllLines(TestFilesDirs[0].GetTestPath("ImsDbTest_expected.csv"));
                 var actual = File.ReadAllLines(outCSV);
                 AssertEx.AreEqual(expected.Length, actual.Length, $"Report difference - expected same line count");
-
-                var i = 0;
-
-                string Localize(string s)
+                for (var i = 1; i < expected.Length; i++) // Skipping header, which may be localized
                 {
-                    return s.Replace(TextUtil.SEPARATOR_CSV, TextUtil.SEPARATOR_CSV_INTL).
-                        Replace(CultureInfo.InvariantCulture.NumberFormat.NumberDecimalSeparator, CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator);
-                }
-
-                foreach (var line in expected)
-                {
-                    var expectedLine = line;
+                    var expectedLine = expected[i];
                     if (CultureInfo.InvariantCulture.NumberFormat.NumberDecimalSeparator !=
                         CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator)
                     {
-                        if (line.StartsWith("\""))
+                        string Localize(string s)
```

Comment:
Let's try for a new utility function in TextUtil. Adding it everywhere is getting ugly.